lib/hutch/worker.rb

```diff
@@ -103,5 +104,11 @@ def error_acknowledgements
     private
 
     attr_accessor :setup_procs
+
+    def unique_consumer_tag
+      prefix = Hutch::Config[:consumer_tag_prefix]
+      unique_part = SecureRandom.uuid
+      "#{prefix}-#{unique_part}"
```

Comment:
I ended up generating this myself. Quote from the commit:

> Due to the interface differences between `bunny` (used for CRuby/MRI)
and `march_hare` (used for JRuby), it's not possible to consistently use
the generator function `queue.channel.generate_consumer_tag` for the
unique part of the consumer tags.

> Instead we can generate the unique part ourselves without relying on
internals of these gems. Using `SecureRandom.uuid` seems like a good
idea.

Are these assumptions valid?